api/src/main/java/com/datatorrent/api/StatsListener.java

```diff
@@ -115,6 +115,28 @@
     List<OperatorResponse> getOperatorResponse();
   }
 
+  /**
+   * An interface to the DAG. Stats listener can get information about
+   * operator or other elements in the DAG through this interface. currerntly
+   * we only provide method to extract the operator name based on the physical
+   * id of the operator. In future more methods can be added.
+   *
+   */
```

Comment:
For the batch scheduling support, having the DAG view available for the operator will be useful. Users should be able to extract the information from the DAG view. Exposing the method for every information may not be ideal.